h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -37,7 +37,8 @@
       "metalearner_fold_column",
       "keep_levelone_frame",
       "metalearner_params",
-      "seed"
+      "seed",
+      "export_checkpoints_dir"
```

Comment:
does it make sense to expose this for SE? I'd rather encourage users to pass it as a metalearner param